nginx.conf

```diff
@@ -0,0 +1,8 @@
+server {
+    # https://stackoverflow.com/a/61753597/7870587
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+        try_files $uri /index.html;
+    }
+}
```

Comment:
In the stackoverflow post, a lot of other things are also present in the nginx.conf file, what about those?
Why are they not important?